dts/msm8916/msm8916-512mb-mtp.dts

```diff
@@ -8,38 +8,21 @@
 / {
 	compatible = "qcom,msm8916-512mb-mtp", "qcom,msm8916";
 	qcom,msm-id = <206 0>;
-	qcom,board-id = <0x08 0x100>;
 	
 	zhihe-ufi-001c {
 		model = "ufi-001b/ufi-001c 4G Modem Stick";
 		compatible = "zhihe,ufi-001c", "qcom,msm8916", "lk2nd,device";
+		qcom,board-id = <8 0x100>;
 
 		/* Register EDL button as HOME button */
 		/* This is the only button on this device */
 		lk2nd,keys = <KEY_HOME 37 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
 	};
 	
-	zhihe-sp970 {
-		model = "sp970 4G Modem Stick";
-		compatible = "zhihe,sp970", "qcom,msm8916", "lk2nd,device";
-
-		/* Register EDL button as HOME button */
-		/* This is the only button on this device */
-		lk2nd,keys = <KEY_HOME 107 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
-	};
-
-	zhihe-uz801 {
-		model = "uz801 4G Modem Stick";
-		compatible = "zhihe,uz801", "qcom,msm8916", "lk2nd,device";
-
-		/* Register EDL button as HOME button */
-		/* This is the only button on this device */
-		lk2nd,keys = <KEY_HOME 23 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
-	};
-
 	zhihe-ufi-16v3 {
 		model = "UFI-16-V3 4G Modem Stick";
 		compatible = "zhihe,ufi-16v3", "qcom,msm8916", "lk2nd,device";
+		qcom,board-id = <11 0x100>;
```

Comment:
I am not sure what `<11 0x100>` stands for, but one thing for sure, 2 properties are common for everything in `msm8916-512mb-mtp.dts`:
- `qcom,msm-id = <206 0>;` 
- `qcom,board-id = <8 0x100>;`

Reply:
> I am not sure what `<11 0x100>` stands for, but one thing for sure, 2 properties are common for everything in `msm8916-512mb-mtp.dts`:
> 
>     * `qcom,msm-id = <206 0>;`
> 
>     * `qcom,board-id = <8 0x100>;`

OK, I have removed all devices except the one(ufi001c) I have. I will open new pull requests if I tested other devices.